lib/incident_io/follow_ups_v2.ex

```diff
@@ -0,0 +1,54 @@
+defmodule IncidentIo.FollowUpsV2 do
+  @moduledoc false
+
+  import IncidentIo
+  alias IncidentIo.Client
+
+  @doc """
+  List all follow-ups for an organisation.
+
+  ## Example
+
+      IncidentIo.FollowUpsV2.list(client)
+
+      IncidentIo.FollowUpsV2.list(client, "some-incident-id")
+
+      IncidentIo.FollowUpsV2.list(client, "some-incident-id", :retrospective)
+
+  More information at: https://api-docs.incident.io/tag/Follow-ups-V2#operation/Follow-ups%20V2_List
+  """
+  @spec list(Client.t()) :: IncidentIo.response()
+  @spec list(Client.t(), binary) :: IncidentIo.response()
+  @spec list(Client.t(), binary, IncidentIo.incident_modes()) :: IncidentIo.response()
```

Comment:
Should change these typespecs to prevent empty strings being passed in.
```suggestion
  @spec list(Client.t(), nonempty_binary) :: IncidentIo.response()
  @spec list(Client.t(), nonempty_binary, IncidentIo.incident_modes()) :: IncidentIo.response()
```